regtests/Dockerfile

```diff
@@ -44,4 +49,39 @@ RUN ./setup.sh
 
 COPY --chown=spark . /home/spark/regtests
 
+# TODO /home/spark/regtests might not be writable in all situations? Seeing those locally :shrug:
```

Comment:
Did you mean to leave this in here? Seems better to not make comment additions that don't have anything to do with the package rename in such a giant PR?

Reply:
Yea - let me recheck that one.

Reply:
Weird.
I run into a file-permission issue when I run the regtests locally via "*podman* compose" - and it *was* the same error in CI. Looks like something has changed in the meantime and CI is happy now w/o this change. :shrug:

Reply:
Removed this change. Let me tackle that in a different PR - definitely cleaner.